apps/wing-console/console/ui/src/services/use-explorer.tsx

```diff
@@ -84,6 +85,32 @@ export const useExplorer = () => {
     setSelectedItems([selectedNode]);
   }, [selectedNode, setSelectedItems]);
 
+  const initialExpandedItems = useMemo(() => {
```

Comment:
instead of creating a memo, you couñd move this code inside the use effect below

Reply:
changed 👍